src/pkg/cli/client/byoc/baseclient.go

```diff
@@ -68,12 +68,17 @@ func NewByocBaseClient(ctx context.Context, grpcClient client.GrpcClient, tenant
 		PulumiStack:   "beta", // TODO: make customizable
 		Quota: quota.Quotas{
 			// These serve mostly to pevent fat-finger errors in the CLI or Compose files
-			Cpus:       16,
-			Gpus:       8,
-			MemoryMiB:  65536,
-			Replicas:   16,
-			Services:   40,
-			ShmSizeMiB: 30720,
+			ServiceQuotas: quota.ServiceQuotas{
+				Cpus:       16,
+				Gpus:       8,
+				MemoryMiB:  65536,
+				Replicas:   16,
+				ShmSizeMiB: 30720,
+			},
+			ConfigCount: 20,   // TODO: add validation for this
+			ConfigSize:  4096, // TODO: add validation for this
+			Ingress:     10,   // TODO: add validation for this
```

Comment:
How do these changes relate to the healthcheck validation?

Reply:
:-/ they don't, but since I need to update the backend to pick up the latest version of this module, I thought I'd pick up other pending changes to this struct. See https://github.com/DefangLabs/defang-mvp/blob/14073858c121be95221a7dd2694c1a5050e21cec/fabric/cmd/server/quota/hosted.go#L7